sdks/python/apache_beam/dataflow_test.py

```diff
@@ -109,11 +109,12 @@ def test_par_do_with_side_input_as_keyword_arg(self):
 
   @attr('ValidatesRunner')
   def test_par_do_with_do_fn_object(self):
-    class SomeDoFn(DoFn):
+    class SomeDoFn(NewDoFn):
       """A custom DoFn for a FlatMap transform."""
 
-      def process(self, context, prefix, suffix):
-        return ['%s-%s-%s' % (prefix, context.element, suffix)]
+      def process(self, prefix, element=NewDoFn.ElementParam,
```

Comment:
Should be differentiate between the deferred vs non-deferred side inputs ? Here I had only added the default value to the deferred one as the other just comes from args